### DISCUSSION TITLE
Inquiry about nnunet Training Issue

### DISCUSSION THREAD

**Discussion by sinner0128** [MIC-DKFZ/nnUNet#1597] [Q&A] [answered]
Dear ,

 My name is Zheng-You, and I am writing to inquire about a recurring issue I have encountered while training nnunet. Occasionally, during the training process, it appears to get stuck, but upon pressing the "enter" key, the training resumes. I have attached a screenshot below, depicting the screen after pressing "enter," where the time appears to have changed significantly. I was wondering if you could kindly provide any recommended solutions or insights to address this matter.

![1691172614661](https://github.com/MIC-DKFZ/nnUNet/assets/110290597/9545a4fa-3e5d-4135-abf2-f99988f33f95)

Here are the specifications of my computer equipment:

CPU: i9-13900K (3GHz / 24 cores)
RAM: 96GB
GPU: RTX4090
I would greatly appreciate any assistance or guidance you can offer regarding this issue. Thank you for your time and attention to this matter.

Looking forward to your prompt response.

Best regards, Zheng-You

**Comment by ancestor-mithril** [comment, marked as answer]
This is related to the Windows command prompt and not to nnUNet. See [this](https://stackoverflow.com/questions/1509816/why-does-selecting-text-in-a-command-prompt-hang-the-running-application).

**Reply by sinner0128** [reply]
Thank you very much for your response. I will look into it and conduct further research. Your assistance is greatly appreciated.
